pulsar-testclient/src/test/resources/perf_client1.conf

```diff
@@ -23,3 +23,5 @@ authParams=myparams
 tlsTrustCertsFilePath=./path
 tlsAllowInsecureConnection=true
 tlsEnableHostnameVerification=true
+proxyServiceURL=https://my-proxy-pulsar:4443/
```

Comment:
this might fail any existing tests which are using this file.

Reply:
The file is used by only one testcase "testReadFromConfigFile" which I have modified to work.